### PR TITLE
[LLM] Mixtral CPU example

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

Mixtral CPU example, based on #9661 and #9671

### 1. Why the change?

Add Mixtral CPU example for SPR

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies
NA

### CODE REVIEW

**Comment by qiyuangong** [comment]
Tested on Xeon Server.

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/mixtral/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install PyTorch CPU as default
+pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
```
Comment (plusbang) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/mixtral/README.md:20]: Maybe we could directly use `pip install --pre --upgrade bigdl-llm[all]` to install.

File: python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### 1. Install
+We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
+
+After installing conda, create a Python environment for BigDL-LLM:
+```bash
+conda create -n llm python=3.9 # recommend to use Python 3.9
+conda activate llm
+
+# below command will install PyTorch CPU as default
+pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
```
Comment (plusbang) [inline python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md:20]: similarly

**Review by plusbang — APPROVED** [review]
Other LGTM

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/mixtral/README.md (hunk 1 above)
Comment (qiyuangong) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/mixtral/README.md:20]: changed to `bigdl-llm[all]`

**Pull request merged**